docs/discussion/DeletingAssertedSubClasses.md

```diff
@@ -0,0 +1,19 @@
+**NOTE** This documentation is incomplete, for now you may be better consulting the [GO Editor Docs](http://wiki.geneontology.org/index.php/Ontology_Editing_Guide)
+
```

Comment:
Rename to "avoid asserting redundant subclasses" and turn this more into an "explanation" piece in the sense of the diataxis framework.